src/jquery.multiselect.js

```diff
@@ -474,7 +477,7 @@
         e.preventDefault();
 
         var $this = $(this);
-        var $inputs = $this.parent().find('input').filter(':visible:not(:disabled)');
+        var $inputs = $this.parent().find('ul input').filter(':visible:not(:disabled)');
```

Comment:
I would just try the following since the ul will be the next sibling of the anchor (no need to find the parent):

`var $inputs = $this.next('ul').find('input').filter(':visible:not(:disabled)');`